specification/app/resource-manager/Microsoft.App/preview/2022-06-01-preview/ManagedEnvironments.json

```diff
@@ -957,6 +1007,63 @@
       "required": [
         "name"
       ]
+    },
+    "workloadProfileStates": {
+      "description": "Collection of all the workload Profile States for a Premium subscription.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Revision resource specific properties",
+          "type": "object",
+          "properties": {
+            "workloadProfileType": {
```

Comment:
Based on spec, this be `name` of the resource.

Reply:
pls re review @jitin